css/modern.css

```diff
@@ -905,6 +966,37 @@ body #right-column-inner a.link-btn {
     border-top: var(--border-size) solid var(--contrast-border) !important;
 }
 
+
+
+.sgy-course-search-form__submit-row {
+    background-color: transparent !important;
+    border-color: transparent !important;
+}
+.sgy-react-select .Select-control, .sgy-react-select .Select-menu-outer {
+    color: var(--text) !important;
+    border-color: transparent !important;
+    border-radius: var(--border-radius) !important;
+}
```

Comment:
A few more borders and backgrounds that are transparent. Can we use `none` instead?